src/styles/card.scss

```diff
@@ -0,0 +1,73 @@
+.card {
+  width: 198px; // including border
+  height: 406px;
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
```

Comment:
For padding and margin values that are vertical or horizontal mirrors, consider using shorthand notation to make the code cleaner. For example, `padding: 32px 19px 40px;` could be `padding: 32px 19px;` if the left and right padding are meant to be the same.